charts/rstudio-workbench/README.md

```diff
@@ -398,37 +388,29 @@ some-key: value1,value2,value3,value4
 This appending/concatenation/array translation behavior only works with the helm chart.
 :::
 
-### Job Json overrides
+## Launcher Templates
 
-If you want to customize the job launch process (i.e., how sessions are defined), edit the following configuration:
+If you want to customize the launcher job templates, use the `launcher.templates` section of the `values.yaml` file. These values are then used within session templates.
 
-- Modify:
-  ```yaml
-  config.profiles.launcher\.kubernetes\.profiles\.conf.<< some selector >>.job-json-overrides`
-  ```
-- Create an array of maps with the following keys:
-  - `target`: The "target" part of the job spec to replace.
-  - `name`: A unique identifier (ideally with no spaces) becomes a configuration filename on disk.
-  - `json`: A YAML value that is translated directly to JSON and injected into the job spec at `target`.
+For example, if you want to add a container image registry credentials secret to allow session images to authenticate to a container registry, you can do so with the following:
 
-Explore the docs in the [Helm repository](https://github.com/rstudio/helm/blob/main/docs/customize.md) for additional information.
+```yaml
+launcher:
+  templateValues:
+    imagePullSecrets:
+    - name: private-registry-creds
+```
+
+For example, if you want to add a toleration to each session, you can do so with the following:
 
 ```yaml
-config:
-  profiles:
-    launcher.kubernetes.profiles.conf:
-      "*":
-        job-json-overrides:
-          - target: "/spec/template/spec/containers/0/imagePullPolicy"
-            json: "Always"
-            name: imagePullPolicy
-          - target: "/spec/template/spec/imagePullSecrets"
-            json:
-              - name: my-pull-secret
-            name: imagePullSecrets
-        container-images:
-          - "one-image:tag"
-          - "two-image:tag
+launcher:
+  templateValues:
```

Comment:
Maybe this is me not understanding Kubernetes, but we call out `launcher.templates` in all the documentation, but all the examples use `templateValues` instead. Is that just how it is?

Reply:
Good catch, it should be `launcher. templateValues` I just pushed the change